src/components/taggable-ui/index.js

```diff
@@ -174,8 +180,9 @@ TaggableUI.propTypes = {
   setSearchString: PropTypes.func,
   queryType: PropTypes.string,
   tagType: PropTypes.string,
-  setTagTypeAndQueryType: PropTypes.func
-
+  setTagTypeAndQueryType: PropTypes.func,
```

Comment:
@jackcarlisle you might want to sub the apostrophe for a [`&#39;`](http://www.freeformatter.com/html-entities.html) (*apostrophe html entity*) on lines 50/51 to avoid syntax highlighter freaking out:
```js
         <h1 className='well problems'>IF YOU&#39;RE HAVING TAG PROBLEMS I FEEL BAD FOR YOU SON <br />
         I&#39;VE GOT 99 PROBLEMS BUT A LINK AIN&#39;T 1</h1>
```
I know it's not quite as snappy, but the effect will be the same in the UI.